.github/workflows/build.yml

```diff
@@ -1,31 +1,37 @@
 name: Build images
 
 on:
-  push:
-    branches:
-      - master
-    tags:
-      - 'v*'
   pull_request:
     branches:
       - master
+  schedule:
+    # Build everyday at 6:00 UTC
+    - cron:  '0 6 * * *'
```

Comment:
What's the advantage of building on a schedule instead of on push? Can we still trigger a new release if we do make changes?

Reply:
The schedule is so we always have the most up to date minor release for Python and Node. Even though we depend on Python 3.9 we want to get 3.9.0, 3.9.1. Same with Node, we might depend on 14 but want to get 14.1, 14.2, etc. In my experience the minor releases never break anything so I'm happy hiding them behind the automation.

I restored the workflow on master on 5cb41b295efe160ad7bac9eb2f72a2efe6ad1b00